### ISSUE TITLE
Why not use the built in JavaScriptCore framework on iOS? + iOS BridgeSupport

### ISSUE THREAD

**Issue by jtreanor** [logancollins/Mocha#30]
I just found this project yesterday and I find it fascinating, in particular the Mocha Interpreter is awesome 😃. I am starting to get my head around it but I have some iOS related questions.

Why is it that there is a JavaScriptCore binary bundled with the project and it doesn't use the build in framework on iOS (7+)?

Also, the usage of BridgeSupport to load OS X frameworks is really cool. Do you think it would be possible to have similar capabilities for iOS frameworks by generating the necessary files with `gen_bridge_metadata`?

Cheers,
James

**Comment by afedor** [comment]
JavaScriptCore isn't included in the 2.0 branch of Mocha, which is a new and improved rewrite and seems to be the one being maintained currently.

**Comment by jtreanor** [comment]
Great, thanks @afedor. I see that now :smile:

**Comment by logancollins** [comment]
Yes, the 1.0 (master) branch is receiving bug fixes, but feature work is currently on the 2.0 branch. Also, the reason for not using the JavaScriptCore from iOS is because it didn't exist prior to iOS 7 (Mocha at one point supported back to 5.0.)
